controllers/kmsissuer_controller.go

```diff
@@ -40,8 +40,6 @@ import (
 const (
 	// DefaultCertDuration is the default duration the CA certificate is valid for.
 	DefaultCertDuration = time.Hour * 24 * 365 * 3 // 3 years
```

Comment:
Is this constant used outside of the package? Does it have to be exported?

Reply:
Made the constants private